projectfiles/CodeBlocks/README.md

```diff
@@ -20,7 +20,13 @@
     Building a 64-bit Wesnoth executable on Windows is currently not supported and will
     fail with the SDK package provided in the next step.
 
-4.  Download the latest `CodeBlocksWinSDK*.zip` package from <http://sourceforge.net/projects/wesnoth/files/SDK/>.
+	To resolve out-of-memory errors when creating debug builds, follow these steps:
+1. If you use 32-bit Windows, run in admin command prompt bcdedit /set IncreaseUserVA 3072
+2. Install MASM32
+3. Run these commands in an admin command prompt:
+cd C:\mingw32\libexec\gcc\i686-w64-mingw32\7.2.0 C:\masm32\bineditbin.exe /LARGEADDRESSAWARE cc1plus.
```

Comment:
1. You're missing the line break between the commands.
2. The .exe extension is missing from cc1plus.exe.
3. Please add \```cmd before the command (in its own line) and ``` after it.